### ISSUE TITLE
all views using current_user need to check for nil

### ISSUE THREAD

**Issue by martindemello** [squush/PairWise#218]
we should be able to display things like standings and pairing without the user being logged in, so calls to `current_user.crosstables_id` etc need to be inside an `<% if current_user %>` block.

**Comment by squush** [comment]
Good point - however I only found one instance where it seemed like this was happening, in the my_tournaments action in the tournaments controller - changed it with my most recent PR, but do you still see other examples I missed?

**Comment by martindemello** [comment]
yes, i found two places where nil errors occasionally got triggered: [here](https://github.com/squush/PairWise/blob/298515370c83fa754732d69228c898d38827a188/app/views/matchups/index.html.erb#L15) and [here](https://github.com/squush/PairWise/blob/5f01955065e46510f1fddd3c3ad8a83aeb7a2dba/app/views/matchups/_matchup.html.erb#L47) - in both cases the view tries to check `current_user.crosstables_id` without first seeing if `current_user` is set or is still nil.

**Comment by martindemello** [comment]
[this](https://github.com/squush/PairWise/blob/fea3ac42fab095c0f11a03c95d2e6d88fd594a76/app/controllers/tournaments_controller.rb#L113) should be `if current_user && current_user.crosstables_id` as well. (the other option is to have a `Guest` user that is always set as the current user by default, so that it can never be nil)
